src/engine/controls/cuecontrol.cpp

```diff
@@ -816,7 +817,7 @@ void CueControl::hotcueActivate(HotcueControl* pControl, double value) {
         }
     }
 
-    m_pHotcueFocus->set(pControl->getHotcueNumber());
+    m_pHotcueFocus->set(pControl->getHotcueIndex() + 1);
```

Comment:
I suggest to define HotcueControl::getHotcueNumber() = HotcueControl::getHotcueIndex() + 1 and use this function consistenly. Please add the documentation about the difference between index and number there and also where to use it.